skietbaan_fe/src/components/ScoreCapture.js

```diff
@@ -441,7 +443,7 @@ export default class search extends Component {
           <div className={this.state.ImageTaken ? "image-container" : "hidden"}>
             <div className="centre-label">
               <div className={!this.state.ImageTaken ? "hidden" : "label-score photo-top-label"}>
-                <b>Score Captured</b>
+                Score Captured
```

Comment:
Put it in an html element

Reply:
what element ?

Reply:
its in a div

Reply:
p element

Reply:
I'm sorry, but I dont understand why

Reply:
The div tag defines the division of a section in an html just like the external wall of your house.
The p tag and other tags are like the walls in your house.
With such, you can independently make modifications of the p tag independently of the div element